### ISSUE TITLE
Consider making Powergate miner index data public

### ISSUE THREAD

**Issue by asutula** [textileio/textile#397]
Slate is asking for a way to get miner index data, maybe we expose the JSON endpoint we have?

**Comment by jsign** [comment]
It's already public: https://pow.hub.textile.io/ 

It's only disabled since it creates load in the Lotus node and messes up with deal throughput and syncing process... also some days ago we had to disable indexes in all hosted pows since there's some weird problem of high mem usage for a particular miner or similar.

So mostly is about:
- Looking at this problem from some days ago.
- Seeing if we can enable the index updating again in the same Lotus node... or maybe running another powergate with its own lotus node only for indexing generation.

**Comment by jsign** [comment]
TL;DR: Need some time dedidcation, but doable with some focus.

**Comment by asutula** [comment]
Sounds good.
